test/integration-test/src/utils.rs

```diff
@@ -0,0 +1,61 @@
+//! Utilities to run tests
+
+use std::{
+    ffi::CString,
+    os::fd::RawFd,
+    sync::atomic::{AtomicU64, Ordering},
+};
+
+use aya::netlink_set_link_up;
+use libc::if_nametoindex;
+use netns_rs::{get_from_current_thread, NetNs};
+
+pub struct Netns {
```

Comment:
let's call this `NetNsGuard` to signal that it has important drop semantics?

Reply:
Sure. Done in 8ec9f73